src/components/Editor/Editor.css

```diff
@@ -22,8 +22,12 @@
   --theme-conditional-breakpoint-color: #ccd1d5;
 }
 
-.out-of-scope .CodeMirror-line, .out-of-scope .CodeMirror-linenumber {
-  opacity: 0.8;
+.paused .in-scope .CodeMirror-line, .paused .in-scope .CodeMirror-linenumber {
+  opacity: 1;
+}
+
+.paused .CodeMirror-line, .paused .CodeMirror-linenumber {
+  opacity: 0.7;
```

Comment:
we're now tracking in-scope and not out-of-scope. 

some reasons:
1. it's easier to get the lines in scope than all the lines that are out of scope
2. it should be faster because there are fewer innies than outies.